com.ampl.ide.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: com.ampl.ide
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: amplide.sh
+tags : 'proprietary'
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+modules:
+  - name: amplide
+    buildsystem: simple
+    build-commands:
+      - install -D amplide.sh /app/bin/amplide.sh
+      - install -Dm 755 apply_extra.sh /app/bin/apply_extra
+      - install -Dp -m 644 com.ampl.ide.desktop /app/share/applications/com.ampl.ide.desktop
+      - install -Dm644 -t /app/share/metainfo com.ampl.ide.metainfo.xml
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/com.ampl.ide.svg
+    sources:
+      - type: extra-data
+        filename: amplide.linux64.tgz
+        only-arches:
+          - x86_64
+        url: https://portal.ampl.com/dl/amplce/amplide.linux64.tgz
```

Comment:
non versioned URL. SO the package installation will break. As for when, next time there is a newer build.

You should at least add the stanza for https://github.com/flathub/flatpak-external-data-checker

Reply:
Maybe i can scrape this page to get the last modified data, but i don't know how. What is the best way?